app/tests/utils/dictCmpTests.py

```diff
@@ -18,9 +18,85 @@ def test_two_flat_dicts_equal(self):
             "project": "lepv"
         }
 
+        dict_3 = {}
+
+        dict_4 = {}
+
+        comp_result_1 = DictUtil.compare(dict_1, dict_2)
+        comp_result_2 = DictUtil.compare(dict_3, dict_4)
+        self.assertEqual(comp_result_1, 0, 'the comparison result of two identical flat dicts should be 0')
+        self.assertEqual(comp_result_2, 0, 'the comparison result of two identical flat dicts should be 0')
+
+    def test_two_flat_dicts_contains(self):
+        
+        dict_1 = {
+            "project1": "lepv1",
+            "project2": "lepv2"
+        }
+        
+        dict_2 = {
+            "project1": "lepv1",
+            "project2": "lepv2", 
+            "project3": "lepv3"
+        }
+
+        dict_3 = {
+            "project1": {"k1": "v1"},
+            "project2": "lepv2"
+        }
+
+        dict_4 = {
+            "project1": {"k1": "v1", "k2": "v2"},
+            "project2": "lepv2"
+        }
+
+        comp_result_1 = DictUtil.compare(dict_2, dict_1)
+        comp_result_2 = DictUtil.compare(dict_4, dict_3)
+        self.assertEqual(comp_result_1, 1, 'the comparison result of two identical flat dicts should be 1')
+        self.assertEqual(comp_result_2, 1, 'the comparison result of two identical flat dicts should be 1')
+
+    def test_two_flat_dicts_contained(self):
```

Comment:
the test ( the method name ) indicates this is about "flat" dicts but this is not true.